openapi/tests/openapi_integration/test_payload_operations.py

```diff
@@ -13,6 +15,26 @@ def setup():
     drop_collection(collection_name=collection_name)
 
 
+def assert_points(points, with_vectors=False):
+    ids = [point['id'] for point in points]
+
+    if not with_vectors:
+        for point in points:
+            point['vector'] = None
+
+    response = request_with_validation(
+        api='/collections/{collection_name}/points',
+        method='POST',
+        path_params={'collection_name': collection_name},
+        body={'ids': ids, 'with_vector': False, 'with_payload': True},
```

Comment:
Should this be

```suggestion
        body={'ids': ids, 'with_vector': with_vectors, 'with_payload': True},
```

instead?

Reply:
Yep :sweat_smile:

Reply:
:smiling_face_with_tear:

Reply:
It's also actually used now. I wanted to use it before for the last test case, but it couldn't yet test for deleted points. I added a `nonexistent_ids` argument, so that it can be used to also check that certain points do _not_ exist and the `with_vectors` argument is now also used.